cosmos-sdk-proto/Cargo.toml

```
@@ -18,6 +18,7 @@ rust-version = "1.72"
 [dependencies]
 prost = { version = "0.13", default-features = false }
 tendermint-proto = "0.39.1"
+serde = "1.0.203"
```

To preserve `no_std` support you'll need to either do this or make `serde` an optional dependency:

```suggestion
serde = { version = "1.0.203", default-features = false }
```

---

Committed this